nearcore/src/config.rs

```diff
@@ -951,6 +1009,9 @@ pub fn init_configs(
     }
 
     let mut config = Config::default();
+    // Make sure node tracks all shards, see
+    // https://github.com/near/nearcore/issues/7388
+    config.tracked_shards = vec![0];
```

Comment:
Because this is the default value for the case of not downloading a config.
The downloaded config should be allowed to override this value.
No change in behavior here.